src/assets/pages/Home.vue

```diff
@@ -0,0 +1,88 @@
+<script setup>
+import { onMounted, ref, watch } from 'vue';
+import CardList from '../../components/Card/CardList.vue';
+import { storeToRefs } from 'pinia';
+import { useSneakersStore } from '../../store/state';
+
+const sneakersStore = useSneakersStore();
+const { items } = storeToRefs(sneakersStore);
+
+watch(
+  () => sneakersStore.items,
```

Comment:
тут може бути `() => items,`
```suggestion
  () => sneakersStore.items,
```